tasks/manage_keys_yml_file.yml

```diff
@@ -99,15 +99,3 @@
         token: "{{ consul_keystore_acl_token }}"
         key: "{{ consul_key_parent + '/' + consul_keystore_properties_key }}"
         value: "{{ consul_to_upload_properties }}"
-
-    - name: Consul | Delete old {{ consul_key_parent }} properties
-      consul_kv:
-        host: "{{ consul_keystore_server }}"
-        port: "{{ consul_keystore_http_port }}"
-        token: "{{ consul_keystore_acl_token }}"
-        key: "{{ consul_key_parent + '/' + key }}"
-        state: absent
-      with_items: "{{ consul_remove_keys }}"
```

Comment:
We should also delete the tasks that generates the variable `consul_remove_keys` since its only purpouse is to be used in this task.

https://github.com/idealista/consul_keystore_role/blob/master/tasks/manage_keys_yml_file.yml#L48-L50

Reply:
There are 2 blocks, first one to process as kv and another one for file. I only delete on file block because as you said in delete block any task define `consul_remove_keys` variable.
On the first block (kv) the variable is defined and as well the delete step block.
When I refactor and add file steps I copy the delete step when obviously it can't work.